data/ocr/label_logo_annotation.txt

```diff
@@ -1,8 +1,107 @@
+1% for the Planet||en:1% for the Planet
+AB||fr:Agriculture Biologique
+Agricoltura Biologica||en:Organic
+agricultura ecologica||en:Organic
+Agricultura Ecologica||en:Organic
+agriculture biologique||fr:Agriculture Biologique
+Agriculture Biologique||fr:Agriculture Biologique
+aop||fr:AOP
+Australian Made||en:Australian Made
+australian made||en:Australian Made
+Bien Se Nourrir||fr:Bien se nourrir
+Bio Siegel||de:Bio Siegel
+bio suisse||en:Bio Suisse
+bio||en:Organic
+Biogarantie||en:Biogarantie
+biogarantie||en:Biogarantie
+Biologische Producten||en:Organic
+canada organic||en:Canada Organic
+certified vegan||en:Vegan
+cosmebio||fr:Cosmébio
+Demeter||en:Demeter
+demeter||en:Demeter
+dlg tele||de:DLG
+DLG||de:DLG
+DLG||de:DLG
+Dolphin Safe||en:Dolphin Safe
+dop vettoriale||en:DOP
+dop||en:DOP
+ecocert||en:Ecocert
+ECOCERT||en:Ecocert
+europees biologisch keurmerk||
+fair trade||en:Fairtrade
+Fairtrade Fortnight 2012||en:Fairtrade
+fairtrade||en:Fairtrade
+Femme Enceinte||en:Not advised for pregnant women
+Forest Stewardship Council||en:FSC
+Forest Stewardship Council||en:FSC
+FSC Mixed Sources||en:FSC
+fsc||en:FSC
+FSC||en:FSC
+German Agricultural Society||de:DLG
+gluten free||en:Gluten-free
+grønt punkt||en:Green Point
+igp vettoriale||en:pgi
+IGP||en:pgi
+igp||en:pgi
+kosher||en:kosher
+label ab||fr:ab-agriculture-biologique
+label rouge vectoriel||en:label-rouge
+Label Rouge||en:label-rouge
+label rouge||en:label-rouge
+lapin de france||fr:Lapin de France
+Le Porc Français||fr:Le Porc Français
+made in france||en:made in france
+made in italy||en:made in italy
+made in luxembourg||en:Made in Luxembourg
+marine stewardship council||en:Marine Stewardship Council
+Marine Stewardship Council||en:Marine Stewardship Council
+max havelaar||en:Max Havelaar
+MSC||en:Marine Stewardship Council
+nature et progrès||fr:Nature et Progrès
+non gmo project||en:Non-GMO project
+not tested on animals||en:Not tested on animals
+Numero Vert||fr:Produit comportant un Numéro Vert
+Ohne Gentechnik||en:Without GMOs
+Organic certification||en:Organic
+Organic farming||en:Organic
+organico||en:Organic
+PEFC||en:PEFC
+pegi 12||en:PEGI
+Pensez Au Tri||fr:Pensez Au Tri
+produit de montagne||fr:produit de montagne
+produit en bretagne||en:produced-in-bretagne
+Produit en Bretagne||en:produced-in-bretagne
```

Comment:
Seems like it should be possible to add some normalization here to lower case all input, remove punctuation etc. such that we don't need to explicitly list all of the possible variations of one label, like `produit en bretagne` vs. `Produit en Bretagne`?